mlflow/alembic/versions/90e64c465722_migrate_user_column_to_tags.py

```diff
@@ -0,0 +1,41 @@
+"""migrate user column to tags
+
+Revision ID: 90e64c465722
+Revises: 451aebb31d03
+Create Date: 2019-05-29 10:43:52.919427
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy import orm
+from mlflow.store.dbmodels.models import SqlRun, SqlTag
+from mlflow.utils.mlflow_tags import MLFLOW_USER
+
+# revision identifiers, used by Alembic.
+revision = '90e64c465722'
+down_revision = '451aebb31d03'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    bind = op.get_bind()
+    session = orm.Session(bind=bind)
+    runs = session.query(SqlRun).all()
+    for run in runs:
+        if not run.user_id:
```

Comment:
Btw, one quirk about this is that if we change the fields in `mlflow.store.dbmodels.models.SqlRun` later on (e.g. if we were ever to remove this `user_id` field) I think this migration would break for old users? Maybe we should inline the desired definition of `SqlRun`, `SqlTag` into the migration file?